### ISSUE TITLE
clever config ?

### ISSUE THREAD

**Issue by rdalverny** [CleverCloud/clever-tools#6]
On first run on a clean install, any invocation of clever will throw an exception, for environment vars and/or configuration file being not defined.

For instance: 
```shell
$ clever -v login
[INFO] [env] API_HOST was not defined, using default
[INFO] [env] LOG_URL was not defined, using default
[INFO] [env] EVENT_URL was not defined, using default
[INFO] [env] OAUTH_CONSUMER_KEY was not defined, using default
[INFO] [env] OAUTH_CONSUMER_SECRET was not defined, using default
[INFO] [env] CONFIGURATION_FILE was not defined, using default
[INFO] [env] CONSOLE_TOKEN_URL was not defined, using default
[INFO] [env] CLEVER_CONFIGURATION_DIR was not defined, using default
[INFO] [env] APP_CONFIGURATION_FILE was not defined, using default
[DEBUG] Load configuration from /Users/rda/.config/clever-cloud
/usr/local/lib/node_modules/clever-tools/node_modules/baconjs/dist/Bacon.js:1257
        throw e;
              ^
TypeError: undefined is not a function
    at /usr/local/lib/node_modules/clever-tools/src/models/configuration.js:45:12
```

It looks like it expects at least an empty JSON hash in the configuration file, a manual `echo "{}" > ~/.config/clever-cloud` fixes the issue.

Thanks!

**Comment by divarvel** [comment]
Thanks for spotting the regression, I'm fixing it

**Comment by divarvel** [comment]
Fixed in 0.3.2
